federation/apis/federation/v1beta1/types.go

```diff
@@ -121,6 +121,8 @@ type ClusterList struct {
 	Items []Cluster `json:"items" protobuf:"bytes,2,rep,name=items"`
 }
 
+type ClusterSelector map[string]string
```

Comment:
Also add documentation. Say that this is expressed as value of annotation for selecting the clusters on which a resource is created